heron/tools/cli/src/python/execute.py

```diff
@@ -147,8 +133,11 @@ def heron_pex(topology_pex, topology_class_name, args=None):
       pex_loader.load_pex(topology_pex)
       topology_class = pex_loader.import_and_get_class(topology_pex, topology_class_name)
       topology_class.write()
-      return Response(Status.Ok)
+      return Response(status=Status.Ok)
     except Exception as ex:
       Log.debug(traceback.format_exc())
-      err_str = "Topology %s failed to be loaded from the given pex" % topology_class_name
-      return Response(Status.HeronError, err_str, str(ex))
+      err_str = "Topology %s failed to be loaded from the given pex: %s" %\
+                (topology_class_name, ex)
+      resp = Response(status=Status.HeronError)
+      resp.add_context(err_str)
+      return err_str
```

Comment:
Why does this return the error string and not the response object containing the error string?